packages/fxa-auth-server/.storybook/preview.js

```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
```

Comment:
It doesn't look like we have the [Controls](https://storybook.js.org/docs/riot/essentials/controls) pane enabled, so these dynamic control matchers might not be neccesary (maybe they were default generated?)